src/main/java/com/doeiqts/pizzavoter/servlet/PizzaVoterServlet.java

```diff
@@ -34,7 +28,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
         if (currentUser != null) {
             // Check for clear command
-            if (currentUser.getNickname().equals("doeiqts")) {
+            if ("doeiqts".equals(currentUser.getNickname())) {
```

Comment:
Just looked and that's a mocking issue. Mocks specifically have things as null that could otherwise never be. I don't think the object I get from Google could actually be null in real life. Don't worry about it, I'll probably just change it when it makes my angry sometime in the future. :P